apps/eaas-api/src/app/services/user.service.ts

```diff
@@ -212,13 +202,12 @@ const searchUsers = async (pattern: string): Promise<Partial<EaasUser>[]> => {
       eaasKnex("users"),
     );
 
-    return await query.select("id", "name", "email");
+    return query.select("id", "email");
   } else {
     // The assumption is that an email should never has spaces, but a name may also have no spaces
     return await eaasKnex("users")
-      .whereRaw("LOWER(name) LIKE ?", `%${pattern.toLowerCase()}%`)
-      .orWhereRaw("LOWER(email) LIKE ?", `%${pattern.toLowerCase()}%`)
-      .select("id", "name", "email", "profile_picture_key");
+      .whereRaw("LOWER(email) LIKE ?", `%${pattern.toLowerCase()}%`)
+      .select("id", "email");
```

Comment:
Are we going to need a searchUsers function?

Reply:
Ah most probably not. We can always bring this back if needed in the future. Gonna remove it